models/index.js

```diff
@@ -13,13 +13,12 @@ if (!global.hasOwnProperty('db')) {
     posts:      sequelize.import(__dirname + '/post')
   };
 
-  /*
-    Associations can be defined here. E.g. like this:
-    global.db.memes.hasMany(global.db.images)
-    global.db.memes.hasMany(global.db.meme_texts)
-    global.db.images.belongsTo(global.db.memes)
-    global.db.meme_texts.belongsTo(global.db.memes)
-  */
+  /* Associations */
+    //global.db.images.hasMany(global.db.memes);
+    global.db.memes.belongsTo(global.db.images,{as:"Images"});
```

Comment:
The association alias here isn't need. I would also recommend renaming the models with capitalized singular names. For example "images" -> "Image".